tests/WE2E/get_expts_status.sh

```diff
@@ -427,7 +427,8 @@ The last ${num_log_lines} lines of the workflow launch log file
     print_info_msg "$msg" >> "${expts_status_fp}"
     tail -n ${num_log_lines} ${launch_wflow_log_fn} >> "${expts_status_fp}" 
   else
-    wflow_status="Workflow status:  NOT LAUNCHED YET"
+    wflow_status="Workflow status:  NOT LAUNCHED YET
+Launching workflow using script \"${launch_wflow_fn}\"..."
```

Comment:
I don't think this is a good idea especially for jenkins tests. This used to be like this before (i.e. launch job after checking status) but I have had issues with jenkins tests, which call this script every minute to check for status of experiments, so I changed it. Moreover, I believe the behavior of this script should be consistent with its name, should simply check status of experiments without launching them. That way it will also respect the launch time assigned in each cronjob.

Reply:
@danielabdi-noaa The original purpose of this script was to get the **updated** status of each experiment in a given directory from the command line without having to go into each directory to call `launch_FV3LAM_wflow.sh`.  I did not realize it had been modified and being used in the Jenkins tests since I've been working on a different branch the past few months.  I'm finding now that I can now no longer use it for its original purpose since the new `run_WE2E_tests.py` script does not call `launch_FV3LAM_wflow.sh` to launch the experiments (so there is no `log.launch_FV3LAM_wflow` log file even if the experiments are running).  I suppose it will once the crontab capability is added back in, but still it would be nice to be able to relaunch a bunch of jobs manually to get their updated workflow statuses.  I'm thinking an easy solution is to add an argument that determines this behavior, say `launch_wflows=[true|false]`, with a default value of `false` so it normally behaves as you expect.  Does that work for you?

Reply:
@gsketefian I disagree on this one. I think it is best to keep modularity and have a script/function that just checks experiment status, and another one that launches them etc. Having been burned with the surprise existence of "launch_wflow" in this script, which was launching the jobs on cheyene after a minute, I am hesitant to bringing back that functionality especially without changing the name of the script.  Also the way it is now, you have no control over when to launch the job (it is immediately launched if doesn't exist), so I doubt the `run_WE2E_tests.py` would use it once it matures. It probably wants to control when to launch jobs and how frequently to check for status using either CRON or something else.

Reply:
@danielabdi-noaa Ok, we can rename it to something like `launch_expts_get_status.sh` and put in that argument (this time with default of "true" makes more sense).  I can also add another argument for a wait period before launching tasks.  `run_WE2E_tests.py` doesn't have to use it.  Will that work?

Reply:
@gsketefian  I still don't think it is a good idea but I will defer to other reviewers. Although the naming change helps, passing flags to make it do everything is not modular programming. I mean it would be odd if I pass a flag to `launch_workflow.sh` to make it check status of an experiment too, wouldn't you agree? In an ideal scenario, there should be one script/function that launches a single workflow, another one that just checks status of that single experiment, another one to iterateve over multiple workflows and launch them, and another to check the status of multiple workflows.  Especially with launching workflows that has more logic (time interval of launch, is it using cron etc), I believe it should be its own function.

Reply:
@danielabdi-noaa, @gsketefian, is there a partial solution to this problem that could be introduced in this PR and then fully implemented through a subsequent PR?  In other words, @gsketefian, is it possible to make a few changes to get your feature working again, but also begin what @danielabdi-noaa has requested?  @danielabdi-noaa, would you be OK with an initial change here, and then a future PR to fully introduce a set of scripts to completely modularize the behavior being discussed?

Reply:
@danielabdi-noaa I added the flag `launch_wflows` as mentioned above.  Adding a flag for a wait time before launching is more difficult; that is really determined by the `CRON_RELAUNCH_INTVL_MNTS` variable in the workflow and/or argument to `run_WE2E_tests.sh` (and soon its python version), and trying to work it into this script would take too much work that is probably not justified as @mkavulich pointed out.  Please let me know if this is a satisfactory solution for the time being.  @mkavulich any thoughts?